File: 프로그래머스/unrated/181891. 순서 바꾸기/순서 바꾸기.js
// 얕은 복사: from 사용
// 배열 합치기: concat 사용
function solution(num_list, n) {
  let arr = Array.from(num_list)

  let arg1 = num_list.splice(0, n)
  let arg2 = arr.splice(n, arr.length)

  return arg2.concat(arg1)
}

File: 프로그래머스/0/181891. 순서 바꾸기/순서 바꾸기.java
class Solution {
  public int[] solution(int[] num_list, int n) {
      int[] result = new int[num_list.length];

      for (int i = n; i < num_list.length + n; i++) {
          int flag = 0;
          if (i >= num_list.length) {
              flag = i - num_list.length;
              result[i - n] = num_list[flag];
          }
          else result[i - n] = num_list[i];
      }

      return result;
  }
}